README.md

```diff
@@ -1,5 +1,18 @@
 # DevOps Apprenticeship: Project Exercise
 
+### Running Tests on Docker
+
+  To Build and Run the Unit & Integration Tests on DockerFile
+  
+    docker build —-target test --tag todo_app:test .
```

Comment:
Minor but caught me out - you've got a slightly different hyphen character here for the first dash in your `--target` flag, which means copy-pasting this command will error

Reply:
Modified Thanks